lxl-web/src/lib/components/DecoratedData.svelte

```diff
@@ -130,16 +124,30 @@
 {#key key}
 	{#if data && typeof data === 'object'}
 		{#if Array.isArray(data)}
-			{#each data as arrayItem}
+			{#if truncate && depth === 1 && data.length > 1}
+				<!-- truncate option; use only first item as data and keep the remainder for tooltip -->
+				{@const [first, ...remainder] = data}
```

Comment:
@johanbissemattsson if you want, take a look at the changes made to `decoratedData` to avoid strange truncate bug. This feels a lot better, not mutating the `data`...

Reply:
This looks much better! 🙌